utils/swift_build_support/swift_build_support/products/swiftpm.py

```diff
@@ -93,8 +94,21 @@ def run_bootstrap_script(self, action, host_target, additional_params=[]):
         # Pass Cross compile host info
         if self.has_cross_compile_hosts(self.args):
             helper_cmd += ['--cross-compile-hosts']
-            for cross_compile_host in self.args.cross_compile_hosts:
-                helper_cmd += [cross_compile_host]
+            if StdlibDeploymentTarget.get_target_for_name(host_target).platform.is_darwin:
+                for cross_compile_host in self.args.cross_compile_hosts:
+                    helper_cmd += [cross_compile_host]
+            else:
+                helper_cmd += [host_target]
+                helper_cmd.append("--skip-cmake-bootstrap")
+                build_toolchain_path = self.install_toolchain_path(host_target,
+                                                                   force_local=True)
```

Comment:
I added this `force_local` flag because when I'm building with a prebuilt toolchain, I want to pass in both the path to the prebuilt toolchain and the path to local toolchain and its resource directory, which in this case contains the Android SDK.